mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/MesosSchedulerClient.java

```diff
@@ -114,4 +138,75 @@ public Subscription sink(@NotNull final Observable<SinkOperation<Send>> spout) {
             .observeOn(Rx.compute())
             .subscribe(subscriber);
     }
+
+    @VisibleForTesting
+    static final class UserAgent {
```

Comment:
I've moved them out now that their access is broader.  Originally I intended for the user to not use these classes but I think it makes sense for them to be accessible now.